Execution/ExecuteBase64DecodedPayload.txt

```diff
@@ -0,0 +1,11 @@
+// Process executed from binary hidden in Base64 encoded file.  Encoding malicious software is a 
```

Comment:
could you please elaborate on the covered tools' commandline (IIUC from your email, it's base64 and Python)